app/views/admin/loans/timeline/table/_timeline.html.slim

```diff
@@ -0,0 +1,11 @@
+.timeline-table
+  - if @loan.root_timeline_entry
+    - if @loan.root_timeline_entry.is_a?(ProjectGroup)
+      table.table
+        = render "admin/loans/timeline/table/group", entry: @loan.root_timeline_entry, is_first: true,
+          max_depth: @loan.root_timeline_entry.max_descendant_depth
+    / If user has created a step through the timeline list, the timeline table is incompatible
+    - elsif @loan.root_timeline_entry.is_a?(ProjectStep)
+      #no-steps-msg.alert.alert-warning This loan is not yet compatible with the timeline table.
+  - else
+    #no-steps-msg.alert.alert-warning = t("project_step.no_steps")
```

Comment:
Warning is perhaps a bit heavy handed here. I'm not sure if an alert is appropriate actually. I'd just put this as plain text for now. Most sites will have a fairly lightweight message in the zero objects case. Just something so that the user doesn't think the page is still loading or something.